hub/api/tests/test_api.py

```diff
@@ -745,8 +745,7 @@ def test_dataset_copy(path, hub_token, num_workers, progress_bar):
         progress_bar=progress_bar,
     )
 
-    assert dest_ds.meta.tensors == ["a", "b", "c", "d"]
-
+    assert dest_ds._all_tensors_filtered(include_hidden=False) == ["a", "b", "c", "d"]
```

Comment:
QQ: do we expect users (e.g. backend) to use `_all_tensors_filtered` or this is only for internal purposes?

Reply:
I think _all_tensors_filtered is only for internal purposes. We can perhaps expose a keys property to the user for this, or maybe there is an existing way. looping in @farizrahman4u

Reply:
ok no need to do changes, just wanted to make sure if this is not intended to be used outside hub repository